### DISCUSSION TITLE
Python Azure function support

### DISCUSSION THREAD

**Discussion by vsfeedback** [Microsoft/PTVS#7627] [Enchancements]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/Python-Azure-function-support/10356893)._

---
Currently only VS Code supports creating Azure Functions in Python; Visual Studio 2022 supports only C#.
It would be really nice if we had Python Azure Function support in VS as well.

---
#### Feedback Bot on 5/7/2023, 09:15 PM: 

(private comment, text removed)

**Comment by StellaHuang95** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.

**Comment by Nickos72** [comment]
guys I have a problem. Can you tell me what is wrong with my code?

import tkinter as tk
import random

def open_new_window():
new_window = tk.Toplevel(root) # Create a new Toplevel window
new_window.title("New Window")
# Add widgets and configure the new window as needed

root = tk.Tk() # Create the main window
root.geometry("500x300")
root.title("Guessing Game")

count = 0

def check_guess():
global count
guess = int(entry.get())
number = random.randint(1, 100)

if guess == number and count != 10:
    tk.Label(root, text="You WON!", font="ar 15").grid(row=3, column=3)   
elif guess < number and count != 10:
    count += 1
    tk.Label(root, text="Higher", font="ar 15").grid(row=3, column=3)
elif guess > number and count != 10:
    count += 1
    tk.Label(root, text="Lower", font="ar 15").grid(row=3, column=3)
elif count == 10:
    tk.Label(root, text="You Lost", font="ar 15").grid(row=3, column=3)
tk.Label(root, text="Say a number from 1 to 100", font="ar 12").grid(row=0, column=3)
tk.Label(root, text="", font="ar 10").grid(row=1, column=3)
tk.Label(root, text="Guess here", font="ar 10").grid(row=2, column=1)

entry = tk.Entry(root)
entry.grid(row=2, column=3)

#Submit button
tk.Button(text="Check guess", command=check_guess).grid(row=2, column=4)

root.mainloop() # Start the main event loop
